src/Helsenorge.Messaging/ServiceBus/Senders/SynchronousSender.cs

```diff
@@ -117,6 +119,9 @@ public async Task<XDocument> SendAsync(ILogger logger, OutgoingMessage message)
                             // update information for existing entry
                             messageEntry.Payload = incomingMessage.Payload;
                             messageEntry.ReplyEnqueuedTimeUtc = incomingMessage.EnqueuedTimeUtc;
+                            //Logs the response time in ms
+                            resonseTime.Stop();
```

Comment:
Hvorfor logger vi responstiden kun her? Vet denne metoden er forferdelig + vanskelig å følge så kan godt hende det er noe jeg ikke helt henger med på

Reply:
Nei, godt spørsmål. Denne implementasjonen logger kun responstid for suksessfulle svarmeldinger slik den er nå, så **ikke** meldinger som er stemplet med timeout. 
Vi bør vel flytte loggingen ut av IF statement? Det vil gjøre at man logger responstid uavhengig av hvor lang tid det tar å få svar. 
@KjetillVassmoLund, er du enig i at man bør logge responstid for begge tilstander?